Chakravarthy7102/marketplace
# feat: Add pick unpick favorite mechanism (#1505)

* feat: Add Pick & Unpick item as favorites mechanism

* feat: Add toasts after picking & unpicking items sagas

* feat: Enable Undo Unpicking

* feat: Toasts with CTAs that can dispatch actions in redux

* fix: Rename a fn with an incorrect name

* style: Use word favorited in test descriptions

* fix: Setting the error to null when starting request actions

* refactor: Remove an unnecessary safe null check

* fix: Use selectors for isPickedByUser and count

* test: Add tests for the onClick mechanism

* refactor: Move the onClick to the bubble inside the favorites counter

* Update webapp/src/modules/vendor/decentraland/favorites/api.spec.ts




* Update webapp/src/modules/vendor/decentraland/favorites/api.spec.ts




* Update webapp/src/modules/favorites/selectors.ts




* fix: Add missing import

* refactor: Use describe each in reducer tests

* feat: If the user closes the login modal the saga should stop

* chore: Upgrade dapps

* test: Add tests for when the favorite data does not exist for the item

* refactor: It is unnecessary to check if the address is connected before returning the fav data in the container

* fix: Add cancel pick action when the user closes the login modal

---------
